editor/questionbankcontroller.js

```diff
@@ -267,7 +268,7 @@ angular.module('createquestionapp', [])
       }
 
       var filterMetaData = {};
-      ecEditor.dispatchEvent('org.ekstep.editcontentmeta:showpopup', { action: 'question-filter-view', subType: 'questions', framework: ecEditor.getContext('framework'), rootOrgId: ecEditor.getContext('channel'), type: 'content', popup: false, metadata: filterMetaData});
+      ecEditor.dispatchEvent('org.ekstep.editcontentmeta:showpopup', { action: 'question-filter-view', subType: 'questions', framework: "NCF", rootOrgId: ecEditor.getContext('channel'), type: 'content', popup: false, metadata: filterMetaData});
```

Comment:
Why have you committed hard-coded values? Please verify your changes before you commit!